### PR TITLE
deposit and withdrawal moved to optimism gateway

### PR DESCRIPTION
**Pull request by dbeal-eth**
there is no need to maintain our own deposit/withdrawal page becuase optimism provides a maintained bridge for us.

this removes all deposit/withdrawal code, and instead links the appropriate points in our UI to their bridge.

### CODE REVIEW

**Review by JChiaramonte7 — COMMENTED** [review]
Makes sense, but I'll leave approvals to anyone more across staking atm

File: constants/links.ts [hunk 1]
```diff
@@ -29,5 +29,9 @@ export const EXTERNAL_LINKS = {
 	},
 	L2: {
 		Optimism: 'https://gateway.optimism.io/welcome',
+		SynthetixDeposit:
+			'https://gateway.optimism.io/?l1Token=0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f&l2Token=0x8700daec35af8ff88c16bdf0418774cb3d7599b4',
```
Comment (0xclem) [inline constants/links.ts:33]: We can probably use snxJS to build the URL, but definitely fine for now

**Review by 0xclem — APPROVED** [review]
Good stuff

**Comment by 0xclem** [comment]
> Makes sense, lgtm, but I'll leave approvals to anyone more across staking atm

True but I can't say no to less code in staking :)

**Pull request merged**